src/pkg/packaging/dir.proj

```diff
@@ -81,14 +81,16 @@
       <templateSvg>$(ProjectDir)resources/images/version_badge.svg</templateSvg>
     </PropertyGroup>
 
+    <MakeDir  Condition="!Exists('$(BaseOutputRootPath)')" Directories="$(BaseOutputRootPath)" />
+    
     <WriteLinesToFile
       File="$(OutputVersionBadge)"
       Lines="$([System.IO.File]::ReadAllText('$(templateSvg)').Replace('ver_number', '$(ProductVersion)'))"
       Overwrite="true"
                       />
   </Target>
   
-  <Target Name="GenerateCompressedFiles" DependsOnTargets="InitPackage;GenerateZip;GenerateTarBall" />
+  <Target Name="GenerateCompressedFiles" DependsOnTargets="InitPackage;GenerateZip;GenerateTarBall" Condition="'$(UsePrebuiltPortableBinaries)' == 'false'"/>
```

Comment:
Yes, post my change, there will be single building of portable binaries and tar , and distro specific native installers will consume the prebuilt portable binaries.

As of today , without my change, portable binaries and tar are built/published in rhel7.2 container. There is no change in building/packaging/publishing orchestration here. My change will be addendum to the existing build steps that consume the portable binaries and generate installers only ( no tars ) for ubuntu and debian. 

Please let me know if my understanding is in in sync with yours.